### PR TITLE
add tooling for invite control

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

**Comment by whyrusleeping** [comment]
hrm... some missing stuff from the lexgen. need to investigate

**Comment by whyrusleeping** [comment]
There we go, still hate the codegen, but now we handle the case where a struct has a field that is a ref of an array of a union... lexicon is great. I love it.

File: cmd/gosky/admin.go [hunk 1]
```diff
@@ -6,10 +6,12 @@ import (
 	"fmt"
 	"os"
 	"sort"
+	"strings"
 	"sync"
 	"time"
 
 	"github.com/bluesky-social/indigo/api"
+	"github.com/bluesky-social/indigo/api/atproto"
 	comatproto "github.com/bluesky-social/indigo/api/atproto"
```
Comment (bnewbold) [inline cmd/gosky/admin.go:15]: duplicate import under a different name?

**Review by bnewbold — APPROVED** [review]
Small note about import duplication, but overall looks good.

Would be good to have specific test coverage for new Lexicon lang features (when we add them to lexgen), but there isn't much a framework for that in place and we will probably re-implement all that at some point anyways.

**Pull request merged**